cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -205,3 +209,23 @@ def _build_spatialized_model_predictions(
             "prediction": aligned_predictions,
         }
     )
+
+
+def _convert_to_h3(df: pd.DataFrame) -> pd.Series:
+    """Builds a Series of h3 indices to the aggregated prediction.
+
+    Args:
+        df: A pandas DataFrame containing the results of
+        _build_spatialized_model_predictions.
+    Returns:
+        A Series of h3 indices to the aggregated prediction for the lat/lon center
+        points which fall within each h3 cell.
+    """
+    df["h3"] = df.apply(_lat_lng_to_h3, axis=1)
+    # TODO: Produce different aggregation(s) depending on prediction type.
+    predictions = df.groupby(["h3"]).prediction.agg("mean")
+    return predictions
+
+
+def _lat_lng_to_h3(row, h3_level=H3_LEVEL):
+    return h3.geo_to_h3(row["lat"], row["lon"], h3_level)
```

Comment:
Where are these lat lon columns expected to come from? They won't be present in the model outputs / predictions.

Reply:
The lat lon columns are from the DataFrame generated after running the model outputs and predictions through _build_spatialized_model_predictions.